db/migrate/20160322224507_make_foreign_key_columns_integers.rb

```diff
@@ -1,6 +0,0 @@
-class MakeForeignKeyColumnsIntegers < ActiveRecord::Migration
-  def change
-    change_column :payment_braintree_customers, :default_payment_method_id, 'integer USING CAST(default_payment_method_id AS integer)'
-    change_column :payment_braintree_transactions, :payment_method_id, 'integer USING CAST(payment_method_id AS integer)'
-  end
-end
```

Comment:
We should drop `card_vault_token` then. `Payment_method_id` is a FK to the payment method tables so it's not a string.